spec/src/main/asciidoc/core/injectionandresolution.asciidoc

```diff
@@ -490,7 +471,6 @@ The log category of a `Logger` depends upon the class of the object into which i
 
 The container must provide a bean with scope `@Dependent`, bean type `InjectionPoint` and qualifier `@Default`, allowing dependent objects, as defined in <<dependent_objects>>, to obtain information about the injection point to which they belong.
 The built-in implementation must be a passivation capable dependency, as defined in <<passivation_capable_dependency>>.
-// TODO move passivation to Full?
```

Comment:
In the "The built-in `Instance`" section below, you drop the same sentence -- presumably to move it to Full. Maybe do the same here?